execution/executor-benchmark/src/db_access.rs

```diff
@@ -70,46 +91,36 @@ impl DbAccessUtil {
         }
     }
 
-    pub fn new_state_key(
-        address: AccountAddress,
-        resource_address: AccountAddress,
-        module: &str,
-        name: &str,
-        type_args: Vec<TypeTag>,
-    ) -> StateKey {
-        StateKey::resource(
-            &address,
-            &Self::new_struct_tag(resource_address, module, name, type_args),
-        )
-        .unwrap()
+    pub fn new_state_key_account(&self, address: &AccountAddress) -> StateKey {
+        StateKey::resource(address, &self.common.account).unwrap()
     }
 
-    pub fn new_state_key_account(address: AccountAddress) -> StateKey {
-        Self::new_state_key(address, AccountAddress::ONE, "account", "Account", vec![])
+    pub fn new_state_key_aptos_coin(&self, address: &AccountAddress) -> StateKey {
+        StateKey::resource(address, &self.common.apt_coin_store).unwrap()
     }
 
-    pub fn new_state_key_aptos_coin(address: AccountAddress) -> StateKey {
-        Self::new_state_key(address, AccountAddress::ONE, "coin", "CoinStore", vec![
-            TypeTag::Struct(Box::new(Self::new_struct_tag(
-                AccountAddress::ONE,
-                "aptos_coin",
-                "AptosCoin",
-                vec![],
-            ))),
-        ])
+    pub fn new_state_key_object_resource_group(&self, address: &AccountAddress) -> StateKey {
+        StateKey::resource_group(address, &self.common.object_group)
     }
 
     pub fn get_account(
         account_key: &StateKey,
         state_view: &impl StateView,
-    ) -> Result<Option<Account>> {
+    ) -> Result<Option<AccountResource>> {
         Self::get_value(account_key, state_view)
     }
 
-    pub fn get_coin_store(
+    pub fn get_fa_store(
+        store_key: &StateKey,
```

Comment:
how does this work?
fa store is in the rg.

Reply:
In db, yes.  But that is StateView's implementation detail - StateView handles the translation